tsdb/store.go

```diff
@@ -237,7 +237,11 @@ func (s *Store) loadShards() error {
 				}
 
 				shard := NewShard(s.databaseIndexes[db], path, s.EngineOptions)
-				shard.Open()
+				err = shard.Open()
+				if err != nil {
+					s.Logger.Printf("failed to open shard %d: %s", shardID, err)
```

Comment:
went for the same error condition as https://github.com/DanielMorsing/influxdb/blob/sharderrcheck/tsdb/store.go#L233